docs/installation.rst

```diff
@@ -121,8 +107,7 @@ Manual Installation
 -------------------
 You can always install the driver directly from a source checkout or tarball.
 When installing manually, ensure the python dependencies are already
-installed. You can find the list of dependencies in
-`requirements.txt <https://github.com/datastax/python-driver/blob/master/requirements.txt>`_.
```

Comment:
why removing this link ? we can just point it to our repo

Reply:
@fruch - I don't see such a document

Reply:
this is the one: https://github.com/scylladb/python-driver/blob/master/requirements.txt